File: src/easy/MaxIceCream.swift
func maxIceCream(_ costs: [Int], _ coins: Int) -> Int {
  var cones = 0
  var arr = costs
  var remaining = coins
  arr.sort()
  for cost in arr {
    if cost > remaining {
      break
    }
    remaining -= cost
    cones += 1

  }
  return cones
}

File: src/easy/max-ice-cream.ts
export const maxIceCream = (costs: number[], coins: number): number => {
	const sorted = [...costs].sort((a, b) => a - b);
	let remaining = coins;
	let cones = 0;
	for (const cost of sorted) {
		if (cost > remaining) break;
		remaining -= cost;
		cones++;
	}
	return cones;
};
